extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcServerProcessor.java

```diff
@@ -0,0 +1,149 @@
+package io.quarkus.grpc.deployment;
+
+import java.lang.reflect.Modifier;
+import java.util.Collection;
+import java.util.List;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.jandex.ClassInfo;
+import org.jboss.logging.Logger;
+
+import io.grpc.internal.DnsNameResolverProvider;
+import io.grpc.internal.PickFirstLoadBalancerProvider;
+import io.grpc.netty.NettyChannelProvider;
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.arc.processor.DotNames;
+import io.quarkus.deployment.IsNormal;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.ServiceStartBuildItem;
+import io.quarkus.deployment.builditem.ShutdownContextBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+import io.quarkus.grpc.runtime.GrpcContainer;
+import io.quarkus.grpc.runtime.GrpcServerRecorder;
+import io.quarkus.grpc.runtime.config.GrpcConfiguration;
+import io.quarkus.grpc.runtime.config.GrpcServerBuildTimeConfig;
+import io.quarkus.grpc.runtime.health.GrpcHealthEndpoint;
+import io.quarkus.grpc.runtime.health.GrpcHealthStorage;
+import io.quarkus.kubernetes.spi.KubernetesPortBuildItem;
+import io.quarkus.smallrye.health.deployment.spi.HealthBuildItem;
+import io.quarkus.vertx.deployment.VertxBuildItem;
+
+public class GrpcServerProcessor {
+
+    private static final Logger logger = Logger.getLogger(GrpcServerProcessor.class);
+
+    public static final String GRPC_SERVER = "/reactive";
```

Comment:
This is incorrect. And should be moved to `FeatureBuildItem`. The client one should be moved to `FeatureBuildItem` too.